src/node/BrowserFetcher.ts

```diff
@@ -27,7 +27,8 @@ import { debug } from '../common/Debug.js';
 import { promisify } from 'util';
 import removeRecursive from 'rimraf';
 import * as URL from 'url';
-import ProxyAgent from 'https-proxy-agent';
+import HttpsProxyAgent from 'https-proxy-agent';
+import { SocksProxyAgent, SocksProxyAgentOptions } from 'socks-proxy-agent';
```

Comment:
Renamed import name since there are different types of proxies now. Also Socks module is having different structure than Https one.